pkgs/servers/caddy/xcaddy/default.nix

```diff
@@ -2,27 +2,34 @@
 
 buildGoModule rec {
   pname = "xcaddy";
-  version = "0.3.1";
+  version = "0.3.2";
 
   subPackages = [ "cmd/xcaddy" ];
 
   src = fetchFromGitHub {
     owner = "caddyserver";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-oGTtS5UlEebIqv4SM4q0YclASJNu8DNOLrGLRRAtkd8=";
+    hash = "sha256-M0eMI/TtUNVqE4F1ifizVb0e4ACGa+yLkG3pJLaaDNs=";
   };
 
   patches = [
+    ./inject_version_info.diff
     ./use_tmpdir_on_darwin.diff
   ];
 
+  ldflags = [
+    "-s"
+    "-w"
```

Comment:
*Technically* this commit does more than fix the build info (strips symbols and DWARF info), but I feel like I'm being too much of a pedant for things we don't have concrete guidelines for, so it's not worth changing -- just figured I'd note it.